habitat-lab/habitat/tasks/rearrange/rearrange_task.py

```diff
@@ -187,9 +187,16 @@ def reset(self, episode: Episode, fetch_observations: bool = True):
             self._is_episode_active = True
 
             if self._should_place_articulated_agent:
+                
+                habitat_config = self._sim.habitat_config
```

Comment:
Don't reference Hydra config in the reset. Instead store as a property in the __init__.